### PR TITLE
feat: movie detail view

### PR DESCRIPTION
**Pull request by cindy-sf**
Movie details view :point_down: 

Light theme 
https://user-images.githubusercontent.com/38938357/126352555-868826b7-b623-4cd8-89a8-73a22a8b63ec.mp4

Dark theme :
https://user-images.githubusercontent.com/38938357/126348668-2cd85f15-59fe-4d6b-9fa0-d110d231566b.mp4

### CODE REVIEW

File: src/views/MovieDetails/types.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  length: number
+  genres: string[]
+  synopsis: string
+  tagline: string
+  language: string
+  notes: Notes
+  followers: 82
+  comments: 34
+  similars: 2
+  characters: 24
```
Comment (j0nathansf) [inline src/views/MovieDetails/types.ts]: Why not have put number  ?

File: src/views/MovieDetails/index.tsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                src={VideoIcon}
+                width={28}
+                height={28}
+              />
+            </TouchableOpacity>
+          </VideoIconWrapper>
+        </MoviePoster>
+        <Text font="POPPINS_SEMI_BOLD" size="BODY_1">{movieInfos.title}</Text>
+        <Text size="BODY_1">{movieInfos.director}</Text>
+        <View style={{ marginBottom: spaces.SMALL, marginLeft: 'auto', marginRight: 'auto' }}>
```
Comment (j0nathansf) [inline src/views/MovieDetails/index.tsx]: You know what to do😉

File: src/views/MovieDetails/index.tsx [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+          ))}
+        </GenderBadgeWrapper>
+        <MovieSpecs
+          duration={movieInfos.length}
+          releaseDate={movieInfos.release_date}
+          language={movieInfos.language}
+        />
+        <Resume>
+          <Text size="SUBTITLE" font="POPPINS_SEMI_BOLD" textAlign="left">Résumé</Text>
+          <View style={{ marginBottom: spaces.XX_SMALL }} />
```
Comment (j0nathansf) [inline src/views/MovieDetails/index.tsx:147]: I accept here because there is only one style

**Review by j0nathansf — CHANGES_REQUESTED** [review]
There are still a lot of things to review in view of the todo present in the code
I invite you to come back to me as soon as it's over

File: src/views/MovieDetails/types.ts (hunk 1 above)
Comment (cindy-sf) [reply src/views/MovieDetails/types.ts]: Well seen, i forgot to remove them 👍

**Review by j0nathansf — APPROVED** [review]

**Pull request merged**